core/src/test/java/org/aerogear/mobile/core/ServiceRegistryTest.java

```diff
@@ -0,0 +1,112 @@
+package org.aerogear.mobile.core;
+
+
+import android.app.Application;
+import android.support.test.filters.SmallTest;
+
+import org.aerogear.mobile.core.configuration.ServiceConfiguration;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.RuntimeEnvironment;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(RobolectricTestRunner.class)
+@SmallTest
+public class ServiceRegistryTest {
+
+    private Application application;
+
+    @Before
+    public void parseMobileCore() {
+        this.application = RuntimeEnvironment.application;
+
+    }
+
+    @Test()
+    public void testSimpleServiceInit() {
```

Comment:
It doesn't make it less clear but IMHO test cases' names would be cleaner w/o the "test" prefix.

Reply:
No they aren't.

"Test" is a verb that makes the method name read like a sentence.

Reply:
You're absolutely right